api/v01/entities/vrp_input.rb

```diff
@@ -176,11 +176,11 @@ module VrpConfiguration
     optional(:vehicle_limit, type: Integer, desc: 'Limit the maxiumum number of vehicles within a solution. Not available with periodic heuristic.')
     optional(:solver_parameter, type: Integer, documentation: { hidden: true }, desc: '[ DEPRECATED : use preprocessing_first_solution_strategy instead ]')
     optional(:solver, type: Boolean, desc: 'Defines if solver should be called')
-    optional(:minimize_days_worked, type: Boolean, default: false, desc: '(Scheduling heuristic only) Starts filling earlier days of the period first and minimizes the total number of days worked. Available only if first_solution_strategy is \'periodic\'. Not available with ORtools.')
-    optional(:same_point_day, type: Boolean, desc: '(Scheduling only) Forces all services with the same point_id to take place on the same days. Available only if first_solution_strategy is \'periodic\'. Not available ORtools.')
-    optional(:allow_partial_assignment, type: Boolean, default: true, desc: '(Scheduling heuristic only) Considers a solution as valid even if only a subset of the visits of a service is performed. If disabled, a service can only appear fully assigned or fully unassigned in the solution. Not available with ORtools.')
+    optional(:minimize_days_worked, type: Boolean, default: false, desc: '(Periodic heuristic only) Starts filling earlier days of the period first and minimizes the total number of days worked. Available only if first_solution_strategy is \'periodic\'. Not available with ORtools.')
```

Comment:
I think `Periodic heuristic only` is more expressive but in the remaining we always use `Periodic only`. Which one do we prefer `Periodic heuristic only` or `Periodic only`? 

Since we are renaming all, may be we can do the rest `Periodic heuristic only` as well?

Reply:
`Periodic only` and `Periodic heuristic only` have a different meaning. 

`Periodic only`means only if you have a schedule. `Periodic heuristic` means only if you have a schedule and you specifically call heuristic.

Reply:
I guess if we have _schedule_ but all service have one visit then we should rather say `Scheduling only`. I am not sure if our customers will understand the difference.

Reply:
I understand better now. I am not sure the difference `Periodic only` and `Periodic heuristic only` is clear enough but I don't have a better suggestion :/ 

Are we renaming `config[:schedule]` into `config[:periodic]` as well? If that is not the case then `Schedule only` should stay as it is, no? 

If the parameter is called Schedule and the heuristic is called Periodic heuristic then  `Periodic heuristic only` and  `Schedule only` are intuitive for me but I am afraid `Periodic only` might be confusing. What do you think @fonsecadeline?

Reply:
You are right ! I guess we can use _Periodic heurisitc_ and _Schedule_only_.
_Periodic only_ is very specific and can be replaced by _Schedule_only_ because in both cases we call ORtools with schedule (only difference is that _Periodic only_ implies that services have more than 1 visit).